From 6b50202f3f11bc5c016e41b8f0c529860a6baabb Mon Sep 17 00:00:00 2001
From: Arun Raghavan <arun.raghavan@collabora.co.uk>
Date: Tue, 15 May 2012 14:13:08 +0530
Subject: doc: Drop 0pointer.de references


diff --git a/configure.ac b/configure.ac
index 69eea2f..3df69d1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -19,7 +19,7 @@
 # along with paprefs. If not, see <http://www.gnu.org/licenses/>.
 
 AC_PREREQ(2.57)
-AC_INIT([paprefs],[0.9.10],[mzcncersf (at) 0pointer (dot) de])
+AC_INIT([paprefs],[0.9.10],[pulseaudio-discuss (at) lists (dot) freedesktop (dot) org])
 AC_CONFIG_SRCDIR([src/paprefs.cc])
 AC_CONFIG_HEADERS([config.h])
 AM_INIT_AUTOMAKE([foreign 1.11 -Wall dist-xz tar-ustar])
diff --git a/doc/README.html.in b/doc/README.html.in
index 50d8676..9bc800f 100644
--- a/doc/README.html.in
+++ b/doc/README.html.in
@@ -60,8 +60,7 @@ href="@PACKAGE_URL@paprefs-0.9.5.tar.gz">Version 0.9.5</a> released; initial rel
 
 <p>PulseAudio Preferences (<tt>paprefs</tt>) is a simple GTK
 based configuration dialog for the <a
-href="http://0pointer.de/lennart/projects/pulseaudio/">PulseAudio
-sound server</a>.</p>
+href="http://pulseaudio.org/">PulseAudio sound server</a>.</p>
 
 <p>Please note that this program can only configure local servers, and
 requires that a special module <tt>module-gconf</tt> is loaded in the
@@ -114,9 +113,9 @@ compilation and <tt>make install</tt> (as root) for installation of
 
 <p>The current release is <a href="@PACKAGE_URL@paprefs-@PACKAGE_VERSION@.tar.gz">@PACKAGE_VERSION@</a></p>
 
-<p>Get <tt>paprefs</tt>'s development sources from the <a href="http://git-scm.org/">GIT</a> <a href="git://git.0pointer.de/paprefs">repository</a> (<a href="http://git.0pointer.de/?p=paprefs.git">gitweb</a>): </p>
+<p>Get <tt>paprefs</tt>'s development sources from the <a href="http://git-scm.org/">GIT</a> <a href="git://anongit.freedesktop.org/pulseaudio/paprefs">repository</a> (<a href="http://cgit.freedesktop.org/pulseaudio/paprefs">web interface</a>): </p>
 
-<pre>git clone git://git.0pointer.de/paprefs</pre>
+<pre>git clone git://anongit.freedesktop.org/pulseaudio/paprefs</pre>
 
 <hr/>
 <address class="grey">Lennart Poettering &lt;@PACKAGE_BUGREPORT@&gt;, September 2009</address>
-- 
cgit v0.10.2

