--- paprefs-0.9.5/src/paprefs.desktop.orig	2006-07-27 20:08:45.000000000 +0200
+++ paprefs-0.9.5/src/paprefs.desktop	2006-09-16 12:42:26.054333250 +0200
@@ -2,10 +2,14 @@
 Version=1.0
 Encoding=UTF-8
 Name=PulseAudio Preferences
+Name[pl]=Ustawienia PulseAudio
 GenericName=Sound server Preferences
+GenericName[pl]=Ustawienia serwera dźwięku
 Comment=View and modify the configuration of the local sound server
+Comment[pl]=Oglądanie i zmiana konfiguracji lokalnego serwera dźwięku
 Exec=paprefs
 Icon=preferences-desktop
 StartupNotify=true
 Type=Application
-Categories=Application;AudioVideo;Audio;
+Categories=GTK;AudioVideo;Audio;
+# vi:encoding=utf-8
